cliquet/resource/__init__.py

```diff
@@ -580,8 +580,17 @@ def postprocess(self, result, action=ACTIONS.READ):
         # when it's added
         if action != ACTIONS.READ:
             # the collection has been changed
+            if not isinstance(result, list):
+                result = [result]
```

Comment:
It should always be a list actually.

When we will plug the notifications with the transaction system, there will multiple creation/updates/deletes. So right now, when a single record is impacted, we put it in a list